solutionbox/code_free_ml/mltoolbox/code_free_ml/analyze.py

```diff
@@ -218,6 +223,7 @@ def _execute_sql(sql, table):
 
   numerical_vocab_stats = {}
   for col_name, transform_set in six.iteritems(inverted_features_target):
+    sys.stdout.write('Analyzing column %s...' % col_name)
```

Comment:
Can we also sys.stdout.flush? Without doing so the message may not appear for a long time.